WordPress/Classes/ViewRelated/Reader/WPImageViewController+Swift.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit
+
+extension WPImageViewController {
+    @objc func loadOriginalImage(for media: Media, success: @escaping (UIImage) -> Void, failure: @escaping (Error) -> Void) {
+        Task {
```

Comment:
Add `@MainActor` to make sure the callbacks are called in the main thread, which is what `ImageLoader` does.

Reply:
I'm not sure why, but it was already calling `success` and `failure` closures on the main thread. `Task` is part of unstructured concurrency. By default, unless you detach it, it runs on the current actor. But in this case, the actor is not specified, so I'm not entirely sure what the documented behavior is. I added `@MainActor` to be explicit and avoid any confusion.